source/stylesheets/base/_navigation.scss

```diff
@@ -12,35 +12,52 @@
   .container {
     display: flex;
     flex-direction: row;
+    position: relative;
 
-    img {
-      transition: height 0.1s ease;
+    .blog-title {
+      color: $red;
+      font-size: 87px;
+      line-height: 80px;
+      margin-bottom: 0.3em;
+      transform: rotate(-8deg);
+      transition: font-size 0.1s ease;
+    }
+
+    .blog-title-first-line {
+      padding-left: 15px;
     }
 
     h3 {
       flex: 1 0 0;
     }
 
     ul {
-      align-content: space-evenly;
-      align-items: flex-end;
-      background: url("../images/nav_background.svg") no-repeat top right;
+      bottom: 20px;
       display: flex;
-      flex: 2 0 0;
-      justify-content: flex-end;
       list-style: none;
+      position: absolute;
+      right: 0;
     }
 
     li {
       a {
+        border-bottom: 0 solid transparent;
         color: $blue;
         font-weight: 400;
-        padding: 3px 20px;
+        padding-bottom: 36px;
+        padding-left: 20px;
+        padding-right: 20px;
```

Comment:
Since you're using Bourbon, you can use the padding mixin for succinctness here. It would be:

```
@include padding(null 20px 36px);
```
https://github.com/thoughtbot/bourbon/blob/master/core/bourbon/library/_padding.scss

Reply:
This is also another place to put your spacing variables to work.